code/modules/reagents/oldchem/reagents/drink/reagents_alcohol.dm

```diff
@@ -17,6 +17,7 @@
 	var/vomit_start = 180	//amount absorbed after which mob starts vomitting
 	var/blur_start = 260	//amount absorbed after which mob starts getting blurred vision
 	var/pass_out = 325	//amount absorbed after which mob starts passing out
+	addiction_chance = 4 //chance to become alcoholic
 
```

Comment:
Hmm, this seems like alcohol will have a 33% chance to cause an addiction, if the person drinks 2 50-u glasses at 10u per sip.